### PR TITLE
Fixed controller loggers

### PR DESCRIPTION
**Pull request by michael81877**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?

Prior to #108, the logs generated by all of the controllers in nifikop looked like so:

```sh
{"level":"info","time":1661343310518261736,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiuser","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiUser","worker count":1}
{"level":"info","time":1661343310519638124,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nificluster","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiCluster","worker count":1}
{"level":"info","time":1661343310519638426,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifidataflow","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiDataflow","worker count":1}
{"level":"info","time":1661343310519685990,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiregistryclient","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiRegistryClient","worker count":1}
{"level":"info","time":1661343310521629398,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nificluster","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiCluster","worker count":1}
{"level":"info","time":1661343310521705585,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343310521721656,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiparametercontext","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiParameterContext","worker count":1}
{"level":"info","time":1661343310522058727,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiusergroup","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiUserGroup","worker count":1}
```

Notice the `controller`, `controllerKind`, and `controllerGroup` are accurate.

After #108, the logs generated by all of the controllers ended up using the same logger (the last one defined in `main.go`):

```sh
{"level":"info","time":1661343860065674615,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860066238015,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860067198746,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860067354869,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860067921263,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860068546926,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860068632782,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
{"level":"info","time":1661343860081604308,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifinodegroupautoscaler","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiNodeGroupAutoscaler","worker count":1}
```

Notice the `controller`, `controllerKind`, and `controllerGroup` are all the exact same. This is wrong, so this PR corrects the logger that all of the controllers use to generate logs.

Basically, the controllers were using the wrong parent logger to create a child loggers for the controllers. This resulted in all of the controllers using the same logger.

### Why?

To have accurate logging from the nifikop controllers.

### Checklist

- [x] Implementation tested
- [x] Error handling code meets the [guideline](docs/error-handling-guide.md)
- [x] Logging code meets the guideline
- [x] User guide and development docs updated (if needed)
- [x] Append changelog with changes

### CODE REVIEW

**Review by r65535 — APPROVED** [review]
LGTM! Tested locally, and logs now display correctly:
```
{"level":"info","time":1661346214703247000,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiparametercontext","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiParameterContext","worker count":1}
{"level":"info","time":1661346214703475000,"caller":"controller/controller.go:227","msg":"Starting workers","controller":"nifiregistryclient","controllerGroup":"nifi.konpyutaika.com","controllerKind":"NifiRegistryClient","worker count":1}

```

**Review by erdrix — APPROVED** [review]
